README.md

```diff
@@ -1 +1,97 @@
-# BookStore
\ No newline at end of file
+# BookStore![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name : BookStore
+
+> The Bookstore is a website similar to the "Awesome Books" website built in the previous module. I will create an MVP version of it that allows you to::
+
+- Display a list of books.
+- Add a book.
+- Remove a selected book.
+
+## Built With
+
+- React JS with Redux
+- JSX
+- JavaScripts
+- GitFlow
+
+## Development set up
+
+Clone Repository using
+
+`git clone git@github.com:Ol-create/book-store.git`
+
+OR using HTTPS
+
+`git clone https://github.com/Ol-create/book-store.git`
+
+move into project directory
+
+`cd bookstore-react`
+
+Install Project dependancies using `NPM`
+
+`npm install`
+
+launch project locally with
+
+`npm start`
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
```

Comment:
- [ ] I think these scripts are not important to be presented in your README file because I think they are just there to help you get started with the `create-react-app` template. Please consider removing them 😃